Makefile

```diff
@@ -85,7 +85,7 @@ endif
 $(addprefix $(.PROXY), $(BIN)): GO_LDFLAGS += -X "$(GOMOD)/internal/version.version=$(VERSION)"
 $(addprefix $(.PROXY), $(BIN)): GO_LDFLAGS += -X "$(GOMOD)/internal/version.commit=$(GIT_SHA)"
 $(addprefix $(.PROXY), $(BIN)): GO_LDFLAGS += -X "$(GOMOD)/internal/version.buildTime=$(shell date)"
-$(addprefix $(.PROXY), $(BIN)): git2go tidy
+$(addprefix $(.PROXY), $(BIN)): tidy
 $(addprefix $(.PROXY), $(BIN)):
 	$(GO) build \
 		-tags static \
```

Comment:
Can be removed

Reply:
I can still see `-tags static` at the latest commit.

Reply:
Ouch, sorry, I was looking at the modified lines 💫 
![image](https://github.com/unikraft/kraftkit/assets/46280801/e726d399-d10d-4ed6-8e7d-17ef62294517)

Reply:
done